tests/sample_data/multimodel_statistics/test_multimodel.py

```diff
@@ -61,14 +65,14 @@ def get_cache_key(value):
     """Get a cache key that is hopefully unique enough for unpickling.
 
     If this doesn't avoid problems with unpickling the cached data,
-    manually clean the pytest cache with the command `pytest --cache-
-    clear`.
+    manually clean the pytest cache with the command `pytest --cache-clear`.
```

Comment:
Just a side comment, was this formatted by yapf? On my end it prefers the left side 🤔

Reply:
It was formatted by me, `yapf` doesn't modify docstrings. I feel that is more readable without the line-break in the middle of the argument.

Reply:
Agreed, but our autoformatters (if not yapf, then maybe docformatter) will prefer the left version so I gave up on changing it back 😅

Reply:
I personally see those more as useful tools to quickly arrive at a decent formatting than the definitive guide on how stuff should be formatted. Hopefully a newer version of docformatter will be smarter.